doc/user/command-line.xml

```diff
@@ -1219,7 +1288,7 @@ vars = Variables('custom.py', ARGUMENTS)
       <para>
 
       &SCons; provides a number of convenience functions
-      that provide ready-made behaviors
+      that provide pre-made behavior definitions
```

Comment:
maybe remove `pre-made` and just leave behavoir definitions.. ? provide pre-made seems a little redundant?

Reply:
Yeah, a "pre-" term is in the section heading already